sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -36,7 +36,7 @@ import (
 // TODO(lostluck): 2018/05/28 Extract these from their enum descriptors in the pipeline_v1 proto
 const (
 	URNImpulse       = "beam:transform:impulse:v1"
-	URNParDo         = "beam:transform:pardo:v1"
+	URNParDo         = "beam:transform:org.apache.beam:pardo:v1"
```

Comment:
I don't think this is a correct URN. These URNs need to match the definitions here: https://github.com/apache/beam/blob/5ef9b7eb433e448b7f0fa37485623df0de6a8eef/model/pipeline/src/main/proto/beam_runner_api.proto#L224

Reply:
Hi @ibzib, the reason that I did in that way is based in this PR https://github.com/apache/beam/pull/15748/files 
This PR Updates cross-language transform URNs to use the new convention and solve "Encountered unsupported logical type URN" I don't know why in my local tests I didn't catch the new failure.
If I use the URN "beam:transform:pardo:v1" I got logical type URN error.
What is the best aprox for this? 
Maybe only fix dataproc creation and deletion and in a future solve each flink tests?
Thanks

Reply:
Agreed with Kyle. The Proto is Canonical, and that PR didn't change the canonical URN for all ParDos. 

The referenced PR was specifically for expansion requests for Cross Language transforms, not DoFns executing in their native environment, which is what this URN represents.

What is the actual bug for "unsupported logical type" that your'e getting? That's how it's reporting it, but is there more to the exception than that?

Reply:
Hi @ibzib I disable the tests that failed and back to the original URN. I think this PR is ready to merge.
A question related with the tracking of the  disabled tests. It's better create new jiras for each error ( at this moment there is two error that affect the tests) and close BEAM-12898 Leaking Vm's or still had open  beam 12898 with a note in the comments that dataproc creation,deletion and fixed vm's is solved and only the tests are missing?

Reply:
After this is merged, we can close BEAM-12898 and create new issues for the test failures.